tests/test_tensor.py

```diff
@@ -56,6 +56,13 @@ def test_ones(backend):
     assert tb.tolist(tb.ones((4, 5))) == [[1.0] * 5] * 4
 
 
+def test_normal(backend):
```

Comment:
this seems to have slipped in but I guess we can just add this test

Reply:
What exactly is this testing though? It just is asserting the call, but not against any target. Is this what we want?

Reply:
added an acfual value to test against